Misc/NEWS.d/next/Library/2024-02-11-07-31-43.gh-issue-82062.eeS6w7.rst

```diff
@@ -0,0 +1,3 @@
+Fix :func:`inspect.signature()` to correctly handle globals in extension
+modules.  Now signatures are correct if globals are use to define default
+values in extension functions and/or methods.
```

Comment:
```suggestion
Fix :func:`inspect.signature()` to correctly handle parameter defaults
on methods in extension modules that use names defined in the module
namespace.
```
Clarify that only methods are changing; functions already worked. Remove the second sentence as it's redundant.